src/main/resources/db/changelog/changeset/01-create-processing-table.yaml

```diff
@@ -0,0 +1,55 @@
+databaseChangeLog:
+  - changeSet:
+      id: 01-create-processing-table
+      author: freesky15
+      changes:
+        - createTable:
+            tableName: processingresults
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: arrivaltasktime
+                  type: timestamp
+                  constraints:
+                    nullable: false
+              - column:
+                  name: taskid
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: solve
+                  type: text
+                  constraints:
+                    nullable: false
+              - column:
+                  name: userid
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: metrics
+                  type: jsonb
+                  constraints:
+                    nullable: true
+              - column:
+                  name: state
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: departureresulttime
+                  type: timestamp
+                  constraints:
+                    nullable: false
+              - column:
+                  name: result
```

Comment:
boolean result looks strange, which kind of result?

Reply:
This field show a status e.g. true - the task was solved, false - the task wasn't solved

Reply:
Is it required if there is a status field which will provide more details, not only true/false?

Reply:
The status field provides the processing status such as "in progress", "finished" etc. The result field provide the result true == success, false == failed.

Reply:
Done!